### PR TITLE
Use set command for sending

### LINKED ISSUE #13: Command configuration is ignored, send and stop commands are hard coded to use sonic_pi
These commands should use the command variable.

### PR DESCRIPTION
**Pull request by lpil**
Fixes #13

### CODE REVIEW

File: plugin/sonicpi.vim [hunk 1]
```diff
@@ -47,13 +47,25 @@ function! s:load_autocomplete()
...
 function! s:load_syntax()
   runtime! syntax/sonicpi.vim
 endfunction
+
+function! s:SonicPiSendBuffer()
+  execute "silent w !" . g:sonicpi_command
+endfunction
+
+function! s:SonicPiStop()
+  call system("sonic_pi stop")
```
Comment (siimphh) [inline plugin/sonicpi.vim]: g:sonicpi_command here as well.

Comment (lpil) [inline plugin/sonicpi.vim]: Thanks. :)

**Comment by lpil** [comment]
Anything you need for this to be merged in @dermusikman ?

**Comment by dermusikman** [comment]
Hey - I'm sorry I've been afk for so long! I'll have a look at this shortly and let you know!

**Comment by dermusikman** [comment]
Thanks so much for the fix!

**Pull request merged**